tests/unit/backend/data/test_DataFactoryService.py

```diff
@@ -321,12 +321,12 @@ def test_deleteWorkspace(self):
         assert self.instance.workspaceDoesExist(wsname)
 
         # won't delete in cis mode
-        Config._config["cis_mode"] = True
+        Config._config["cis_mode"]["enabled"] = True
```

Comment:
these should be refactored to use [config_override](https://github.com/neutrons/SNAPRed/blob/39b7ef6eb45c8062c3b74f6c2b88beb4e8d876e2/tests/util/Config_helpers.py#L21)